packages/react/src/hooks/constants.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  SendUserAttributeVerificationCodeOutput,
+  UpdateUserAttributesOutput,
+  UserAttributeKey,
+} from "aws-amplify/auth";
+
+export type UserAttributes = Record<UserAttributeKey, string | undefined>;
+
+export const DefaultAttributes: UserAttributes = {
+  email: undefined,
+  phone_number: undefined,
+  address: undefined,
+  birthdate: undefined,
+  email_verified: undefined,
+  family_name: undefined,
+  gender: undefined,
+  given_name: undefined,
+  locale: undefined,
+  middle_name: undefined,
+  name: undefined,
+  nickname: undefined,
+  phone_number_verified: undefined,
+  picture: undefined,
+  preferred_username: undefined,
+  profile: undefined,
+  sub: undefined,
+  updated_at: undefined,
+  website: undefined,
+  zoneinfo: undefined,
+};
+
+// Default object to be passed into useDataState
+export const defaultUpdateUserAttributesOutput: UpdateUserAttributesOutput = {
+  ["custom:placeholder"]: {
```

Comment:
What is this for?

Reply:
Originally I'd seen a type error if I didn't include at least one custom attribute on the defaultUpdate type, but now I've removed the custom one. Looking to the comment below too, I've removed the codeDeliveryDetails because it's not required. One weird thing still is that updateUserAttributes only returns the attributes and the nextStep for the attributes you submitted, but if I don't include every standard attribute in the defaultUpdateOutput I see type errors.